src/cart/multi-node-test.sh

```diff
@@ -1,196 +0,0 @@
-#!/bin/bash
```

Comment:
Why was this file removed?

Reply:
I think it's stale and not doing anything.